sdk/iotoperations/arm-iotoperations/tsp-location.yaml

```diff
@@ -1,4 +1,4 @@
 directory: specification/iotoperations/IoTOperations.Management
-commit: 7881f98bf6f1d2e4943f0eb25e6b053c73380aa0
+commit: e725ba2ceffe71772df6918b2950bf571577f968
```

Comment:
could you ensure the commit id is in main?

Reply:
yes